apps/dashboard/app/views/dashboard/_xdmod_widget_jobs.html.erb

```diff
@@ -1,92 +1,73 @@
-      </tr>
-    </thead>
-    <tbody>
-      {{#each results}}
-      <tr>
-        <td>{{name}}</td>
-        <td><a target="_blank" href="{{job_url}}">{{local_job_id}}&nbsp;<i style="display: inline" class="fa fa-external-link-square-alt"></i></a></td>
-        <td>{{resource}}</td>
-        <td>{{start_time}}</td>
-        <td>{{end_time}}</td>
-        <td>{{walltime}}</td>
-        <td>{{cpu_user}}</td>
-      </tr>
-      {{else}}
-      <tr><td colspan="7">No data available.</td></tr>
-      {{/each}}
-    </tbody>
-  </table>
-
   {{#if error}}
-    <div class="alert alert-danger">{{error}}</div>
-  {{/if}}
-
+    <div class="panel-body">
+      <div class="alert alert-danger">{{error}} Please ensure you are <a href="{{xdmod_url}}">logged into XDMoD first</a>, and then try again.</div>
+    </div>
+  {{else}}
   {{#if loading}}
-    <div class="alert">LOADING...</div>
-  {{/if}}
+    <div class="panel-body">
+      <p>LOADING...</p>
+    </div>
+  {{else}}
+    <div class="panel-body">
+      <div class="table-responsive">
+        <table class="table table-striped table-condensed">
+          <thead>
+            <tr>
+              <th>ID</th>
+              <th>Name</th>
+              <th>Date</th>
+              <th>CPU</th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#each results}}
+            <tr title="{{job_name}} - {{local_job_id}}" data-toggle="tooltip" data-placement="left">
+              <td style="white-space: no-wrap"><a target="_blank" href="{{job_url}}">{{local_job_id}}&nbsp;<span class="fa fa-external-link-square-alt"></span></a></td>
+              <td style="max-width: 150px; text-overflow: ellipsis; overflow: hidden;">{{job_name}}</td>
+              <td>{{date}}</td>
+              <td>{{cpu_label cpu_user}}</td>
+            </tr>
+            {{else}}
+            <tr><td colspan="7">No data available.</td></tr>
+            {{/each}}
+          </tbody>
+        </table>
+      </div>
+    </div>
 
-</div></div>
+    <div class="panel-footer">
+      Showing first {{page_limit}} of {{totalCount}} jobs. See <a href="{{xdmod_url}}">your XDMoD dashboard&nbsp;<span class="fa fa-external-link-square-alt"></span></a> for more information.
+    </div>
+  {{/if}}
+  {{/if}}
+</div>
 </script>
 
+
 <script>
 (function(){
 
- var startOfYear = '<%= Date.today.beginning_of_year.strftime("%Y-%m-%d") %>',
+var startOfYear = '<%= Date.today.beginning_of_year.strftime("%Y-%m-%d") %>',
      thirtyDaysAgo = '<%= 30.days.ago.strftime("%Y-%m-%d") %>',
      today = '<%= Date.today.strftime("%Y-%m-%d") %>';
 
+var jobsPageLimit = 10;
+
 var jobsUrl = new URL('<%= Configuration.xdmod_host %>/rest/v1/warehouse/search/jobs');
 jobsUrl.searchParams.set('_dc', Date.now());
 jobsUrl.searchParams.set('start_date', thirtyDaysAgo);
 jobsUrl.searchParams.set('end_date', today);
-jobsUrl.searchParams.set('realm', 'Jobs');
-jobsUrl.searchParams.set('limit', 10);
+jobsUrl.searchParams.set('realm', 'SUPREMM');
```

Comment:
I don't have access to the `SUPREMM` realm, so I can't see recently completed jobs.

Reply:
The SUPREMM realm is tied to your user account.  Staff (who are flagged by someone with admin rights) will see all jobs in SUPREMM realm, everyone else only sees their own jobs.  So care must be taken if we are searching SUPREMM realm to only show jobs belonging to the actual user.

Reply:
Specifically the error I get is: `Request for unknown Realm: supremm` so I don't even know about the realm.  (the request was all uppercase SUPPREM where the response lowercase, in case that's in question)

Reply:
Ill open a separate issue. Looks like the api response for current user information from XDMoD will be getting more information soon.